lambda/save-raw-events/save-raw-events.ts

```diff
@@ -37,16 +37,19 @@ export const getTTLDate = (): number => {
 };
 
 export const validateUser = (user: UserData): void => {
-  if (user.user_id === undefined) {
+  if (user.user_id === undefined || user.user_id === null) {
```

Comment:
```suggestion
if (!user.user_id) {
'''

Reply:
Amazing, much better cheers for the improvement @TaherKapasi that's way less verbose!